diff/keys_test.go

```diff
@@ -51,7 +56,9 @@ func TestDiff(t *testing.T) {
 	}
 
 	result := CompareConfigs(g1, g2, opts...)
-	assert.Equal(t, expected, result)
+	if !reflect.DeepEqual(expected, result) {
```

Comment:
here we might prefer go-cmp over reflect.DeepEqual :)

Reply:
I'm really loathe to bring in things out of the standard library, `spew` is my only real vice. Are you OK with just using `reflect`?